src/collections/_documentation/platforms/javascript/index.md

```diff
@@ -61,7 +61,51 @@ Then, you can see the error in your dashboard:
 [{% asset js-index/error-message.png alt="Error in Unresolved Issues with title This is my fake error message" %}]({% asset js-index/error-message.png @path %})
 
 ## Capturing Errors
-In most situations, you can capture errors automatically with `captureException()`.
+Automatically and manually capture errors, exceptions, and rejections.
+
+### Automatically Capture Errors
+By including and configuring Sentry, the SDK will automatically attach global handlers to capture uncaught exceptions and unhandled rejections.
+
+[{% asset js-index/automatically-capture-errors.png alt="Stack trace of a captured error" %}]({% asset js-index/automatically-capture-errors.png @path %})
+
+Browsers take security measures when serving script files from different origins. To ensure errors are routed to Sentry, configure the CORS headers, and add script attributes. 
+
+**CORS Attributes and Headers**
+
+To gain visibility into a JavaScript exception thrown from scripts originating from different origins, do two things:
```

Comment:
So, I think this is too much information for the docs at this stage. Like, 95-99% of users won't have to deal with this at all. It should be reserved for a troubleshooting section.